### ISSUE TITLE
multipart/form-data automated tests not working

### ISSUE THREAD

**Issue by acrontum-sanchez** [acr-lfr/generate-it-typescript-server#164]
When adding a route like mention on [https://github.com/acr-lfr/generate-it-typescript-server/issues/162](https://github.com/acr-lfr/generate-it-typescript-server/issues/162)
the server generation fails.

The only workaround was to delete the autogenerated tests and remove 

```
    "tests": {
      "outDir": "src/domains/__tests__"
    },
```

from nodegenrc config.

Generate it version: 5.33.0
Swagger 3

**Comment by p-mcgowan** [comment]
yup - aitogen tests assume oa2 "in":"formData" format so need to rework them to handle oa3 cintent type style.
might tack onto auto gen test rewerite tomorrow
